### DISCUSSION TITLE
Mi Wireframe

### DISCUSSION THREAD

**Discussion by Exusai** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#90] [Show and tell]
Hola Explorers, me gustaría compartir mi avance, este es el wireframe que hice:
![image](https://user-images.githubusercontent.com/47704357/155390456-82f38fa9-3b66-4026-96fa-afd3b3a4deb4.png)
![image](https://user-images.githubusercontent.com/47704357/155390550-b74a3937-2d4f-4bbf-94ce-974f02651dcb.png)

pueden visitarlo de forma interactiva en este link:
https://xd.adobe.com/view/4210832a-d35a-4c65-bd6a-a52bfcf9884d-1743/

Si puedo ayudarlos con algo, será un gusto.

**Comment by Yvillacis** [comment]
Faber, me encantaron tus disenos, que herramienta usaste para generar esto?

**Reply by Exusai** [reply]
Es Adobe XD

**Comment by JuanSalvador73** [comment]
Genial, ¿que software usaste para el diagrama? Porque la mayoría de los que hay online tienen el sistema gratuito limitado en recursos y proyectos.

**Reply by Exusai** [reply]
Es AdobeXD

**Comment by RodolfoMorquecho** [comment]
Te quedo muy bien, por cierto la parte de UI tambien la haras/hiciste en adobe?

**Reply by Exusai** [reply]
Si, aún no empiezo, pero la trabajare en el Adobe XD

**Reply by RodolfoMorquecho** [reply]
Seguro estarán igual de geniales, éxito!
